js/scripts.js

```diff
@@ -56,70 +83,65 @@ let pokemonRepository = (function () {
       return response.json();
     }).then(function (details) {
       // Adding the details to the item
-      item.imageUrl = details.sprites.front_default;
+      item.imageUrl = details.sprites.other.home.front_default;
       item.height = details.height;
       item.types = details.types;
+      item.abilities = details.abilities;
     }).catch(function (e) {
       console.error(e);
     });
   }
-
-  function showDetails(pokemon) { //creating function to show more details about pokemon when clicked
+  //creating function to show more details about pokemon when clicked
+  function showDetails(pokemon) { 
     loadDetails(pokemon).then(function() {
       showModal(pokemon);
     });
   }
 
   //showModal function
-  let modalContainer = document.querySelector('#modal-container');
+  let modalContainer = document.querySelector('#pokemonModal');
 
   function showModal(pokemon) {
-    modalContainer.innerHTML = '';
-    let modal = document.createElement('div');
-    modal.classList.add('modal');
+    let modalTitle = $('.modal-title');
+    let modalBody = $('.modal-body');
 
-    let closeButtonElement = document.createElement('button');
-    closeButtonElement.classList.add('modal-close');
-    closeButtonElement.innerText = 'Close';
-    closeButtonElement.addEventListener('click', hideModal);
+    modalTitle.empty();
+    modalBody.empty();
 
-    let titleElement = document.createElement('h1');
-    titleElement.innerText = pokemon.name.toUpperCase();
+    let titleElement = document.createElement('h5');
+    titleElement.innerText = pokemon.name;
 
     let heightElement = document.createElement('p');
-    heightElement.innerText = 'Height: ' + pokemon.height;
+    heightElement.innerText = 'Height: ' + pokemon.height + 'm';
 
-    let imageElement = document.createElement('img');
-    imageElement.classList.add('pokeImage');
-    imageElement.src = pokemon.imageUrl;
+    let pokemonTypes = [];
+    Object.keys(pokemon.types).forEach(key => {
+      pokemonTypes.push(' ' + pokemon.types[key].type.name);
+    });
 
-    modal.appendChild(closeButtonElement);
-    modal.appendChild(titleElement);
-    modal.appendChild(heightElement);
-    modal.appendChild(imageElement);
-    modalContainer.appendChild(modal);
+    let typesElement = document.createElement('p');
+    typesElement.innerText = 'Types: ' + ' ' + pokemonTypes;
+    typesElement.classList.add('types-element');
 
-    modalContainer.classList.add('is-visible');
-  }
+    let pokemonAbilities = [];
+    Object.keys(pokemon.abilities).forEach(key => {
+      pokemonAbilities.push(' ' + pokemon.abilities[key].ability.name);
+    });
 
-  function hideModal() {
-    modalContainer.classList.remove('is-visible');
-  }
+    let abilitiesElement = document.createElement('p');
+    abilitiesElement.innerText = 'Abilities: ' + pokemonAbilities;
+    abilitiesElement.classList.add('abilities-element');
```

Comment:
We can improve this using list items as well with abilities as the section title. Same goes for the types. If you want to but this is fine as well